circle.yml

```diff
@@ -9,7 +9,12 @@ machine:
     services:
         - docker
     environment:
-        CXX: G++-4.9
+        CXX: g++-4.9
+
+dependencies:
+    override:
```

Comment:
Cool, moved only removal of node_modules to "pre". Also realized dependencies build had a few env var exports, so ya, shouldn't use "override"